desktop/package.json

```diff
@@ -86,5 +88,8 @@
     "webpack-dev-server": "1.14.1",
     "webpack-hot-middleware": "2.10.0",
     "webpack-target-electron-renderer": "0.4.0"
+  },
+  "optionalDependencies": {
+    "fsevents": "1.0.11"
```

Comment:
Yes this appears to be the case:

```
keybase@1.0.1 /Users/mgood/Documents/go/src/github.com/keybase/client/desktop
└─┬ babel-cli@6.7.7
  └─┬ chokidar@1.4.3
    └── fsevents@1.0.11 
```

When I last tried this, the Travis `npm install` resulted in an error. I'm re-running an install without the optional dependency in `package.json`, let's see if it still causes issues.